src/cls/IPM/Storage/Module.cls

```diff
@@ -350,7 +362,20 @@ ClassMethod ExecutePhases(pModuleName As %String, pPhases As %List, pIsComplete
         }
         Quit:$$$ISERR(tSC)
 
-        If 'tIsCustomPhase {
+        If tIsCustomPhase {
+			// Handle CPF merges in custom phases
```

Comment:
Nit: Remove this comment since it isn't specific to CPF merges necessarily.
Also, in a few of these files, the spacing seems off. Not sure if that has to do with github though.